lib/manager/cargo/artifacts.spec.ts

```diff
@@ -67,9 +78,9 @@ describe('manager/cargo/artifacts', () => {
   });
   it('returns null if unchanged', async () => {
     fs.stat.mockResolvedValueOnce({ name: 'Cargo.lock' } as any);
-    fs.readFile.mockResolvedValueOnce('Current Cargo.lock' as any);
+    readLocalFile.mockResolvedValueOnce('Current Cargo.lock' as any);
```

Comment:
use imported fs from test-utils

Reply:
Hi @viceice I tried to keep `fs-extra` unmocked as you suggested, but I had to mock it again since I needed to mock return value of `fsExtra.stat`. Could you please review the changes for the `lib/manager/cargo/artifacts.spec.ts` only and tell me if they are OK and I can proceed with the rest of files?